### DISCUSSION TITLE
lilygo T-can485 ?

### DISCUSSION THREAD

**Discussion by affer** [txubelaxu/esphome-jk-bms#38] [Q&A] [answered]
can i use this board? specific the 485 port. i have 6 JKBMS PB

**Comment by txubelaxu** [comment]
I thonk @Imanol is using this board.

**Comment by Sleeper85** [comment, marked as answer]
> can i use this board? specific the 485 port. i have 6 JKBMS PB

Yes it works RS485 and CAN, tested with YamBMS.

**Reply by affer** [reply]
Yes - i wrote this before i found YamBMS - u helped me a lot :-d Thanks

**Reply by Sleeper85** [reply]
Oh yeah! I didn't pay attention to your nickname :smile:
